### PR TITLE
CONSOLE-2773: Consolidate E-I namespaces

### PR DESCRIPTION
**Pull request by zherman0**
[CONSOLE-2773](https://issues.redhat.com/browse/CONSOLE-2773)
Consolidate E-I namespaces

/hold until PR8279 merges

### CODE REVIEW

**Comment by zherman0** [comment]
Accidentally combined PR8279  branch.  Need to rebase after 8279 merges.

**Comment by spadgett** [comment]
@zherman0 Do you mind rebasing before review? There are a number of conflicts.

**Review by spadgett — APPROVED** [review]
/lgtm
/hold for approvals
/assign @yapei @ahardin-rh @sferich888 

This should have no outward changes, just consolidating the messages into fewer files to improve console load time. Note that this means the translations won't appear until after we get the next translation drop back due to the reorganization. @yapei You will have to use pseudolocalizations to test.

**Comment by spadgett** [comment]
@rh-tchuang Any concerns with this? We're trying to consolidate the messages into fewer files. These strings will appear to be untranslated for the next drop. My understanding is that with translation memory this shouldn't be an issue, but if it is, let us know!

cc @rebeccaalpert

**Comment by yapei** [comment]
@XiyunZhao  Please help test this PR

**Comment by XiyunZhao** [comment]
No issue found on regression testing, and all case is being tested on pseudolocalizations

**Comment by spadgett** [comment]
/lgtm cancel

See https://github.com/openshift/console/pull/8279#issuecomment-804898197

**Comment by XiyunZhao** [comment]
@spadgett I saw there are some changes on file of 'duration-hook.ts' for 1/6/24 hours, but when checking the duration of Cluster utilization part on Home-Overview page is not translate. please guide me if it is correct

**Comment by XiyunZhao** [comment]
@zherman0 any update for the issue?

**Comment by rh-tchuang** [comment]
> @rh-tchuang Any concerns with this? We're trying to consolidate the messages into fewer files. These strings will appear to be untranslated for the next drop. My understanding is that with translation memory this shouldn't be an issue, but if it is, let us know!
> 
> cc @rebeccaalpert

Hey @spadgett 
Yup this won't be too much of an issue; in fact consolidating messages is good news for me, currently we are uploading 97 files per language and it can slow down the UI dramatically. You're right with the TM; but I'll need to change a couple of segmentation rules during file importation. No big deal :)

**Comment by XiyunZhao** [comment]
@zherman0 The issue that I reported can be fixed in other PR, please help to resolve the conflict file, and I would like to double check it on the cluster which start by boot

**Comment by spadgett** [comment]
Hey @zherman0, it looks like this PR still needs the namespace updates so that the namespace matches the package name.

File: frontend/public/i18n.js
Comment (dtaylor113) [inline frontend/public/i18n.js:36]: Hi, why are we removing namespaces outside of E-I?

File: frontend/packages/pipelines-plugin/integration-tests/support/page-objects/pipelines-po.ts
Comment (dtaylor113) [inline frontend/packages/pipelines-plugin/integration-tests/support/page-objects/pipelines-po.ts:80]: Hi is this an "E-I" change?  'pipeline' and 'details-page'?

**Review by dtaylor113 — APPROVED** [review]
/lgtm

File: frontend/public/i18n.js
Comment (zherman0) [reply frontend/public/i18n.js:36]: I did not remove these files in the a-d so just did it here.

**Review by spadgett — APPROVED** [review]
/lgtm

👍

**Pull request merged**
